crates/bitwarden-wasm/src/client.rs

```diff
@@ -54,3 +55,28 @@ impl BitwardenClient {
         })
     }
 }
+
+#[wasm_bindgen]
+pub fn argon2(
+    password: &[u8],
+    salt: &[u8],
+    iterations: u32,
+    memory: u32,
+    parallelism: u32,
+) -> Vec<u8> {
+    let argon = Argon2::new(
+        Algorithm::Argon2id,
+        Version::V0x13,
+        Params::new(
+            memory * 1024, // Convert MiB to KiB
+            iterations,
+            parallelism,
+            Some(32),
+        )
+        .unwrap(),
+    );
+
+    let mut hash = [0u8; 32];
+    argon.hash_password_into(password, salt, &mut hash).unwrap();
+    hash.to_vec()
```

Comment:
@Hinton Changing this to Vec<u8> generates the Uint8Array bindings and works correctly, as you mentioned in the comment